datastore/src/main/java/io/spine/server/storage/datastore/query/ToRecords.java

```diff
@@ -66,12 +66,13 @@
 
     @Override
     public Iterable<R> apply(IntermediateResult result) {
-        List<@Nullable Entity> entities = result.entities();
-        Stream<Entity> stream =
+        var entities = result.entities();
+        var stream =
                 entities.stream()
                         .filter(Objects::nonNull);
         stream = filter(stream);
-        ImmutableList<R> records =
+        @SuppressWarnings("ConstantConditions") /* `null` were already filtered out. */
+        var records =
                 stream.map(this::toRecord)
```

Comment:
Same in this file.

Reply:
Moved to a single line: 
```java
 entities.stream().filter(Objects::nonNull);
```

As there is another `.filter(..)` on the next line.